.github/workflows/production-tag.yml

```diff
@@ -0,0 +1,128 @@
+name: Create Production Tag
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ['Continuous Integration']
+    types: [completed]
+    branches: [main]
+
+concurrency:
+  group: production-tag
+  cancel-in-progress: true
+
+env:
+  SLACK_CHANNEL: C06DSBT7CBW #status-next-build
+  DSVA_SCHEDULE_ENABLED: true
+
+jobs:
+  create-production-tag:
+    name: Create Production Tag
+    runs-on: ubuntu-latest
+    # Run the workflow unless it was triggered by CI and that failed
+    if: ${{ !(github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'failure') }}
+    outputs:
+      RELEASE_NAME: ${{ steps.export-release-name.outputs.RELEASE_NAME }}
+    permissions:
+      id-token: write
+      contents: write
+
+    steps:
+      - name: Cancel workflow due to DSVA schedule, unless this is a manual build
+        if: ${{ github.event_name != 'workflow_dispatch' && env.DSVA_SCHEDULE_ENABLED != 'true' }}
+        uses: andymckay/cancel-action@b9280e3f8986d7a8e91c7462efc0fa318010c8b1 # v0.3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # v2.0.0
+        with:
+          role-to-assume: ${{ vars.AWS_ASSUME_ROLE }}
+          aws-region: us-gov-west-1
+
+      - name: Get bot token from Parameter Store
+        uses: department-of-veterans-affairs/action-inject-ssm-secrets@d8e6de3bde4dd728c9d732baef58b3c854b8c4bb # latest
+        with:
+          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
+
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install Npm Dependencies
+        run: |
+          yarn set version 3.6.1
+          HUSKY=0 yarn install --immutable
+
+      - name: Get current ref
+        id: get-current-ref
+        run: echo REF=$(git rev-parse HEAD) >> $GITHUB_OUTPUT
+
+      - name: Get latest tag
+        id: get-latest-tag
+        run: echo LATEST_TAG_VERSION=$(git fetch --all --tags > /dev/null && git tag -l | sort -V --reverse | head -n 1) >> $GITHUB_OUTPUT
+
+      - name: Get next tag version
```

Comment:
This was reworked some from the Content Build version, which assumes the numbering scheme will always be `v0.0.123` or some other number on the end. The rework expects a semver-type version (3 numbers separated by `.` but otherwise makes no assumptions about the incoming tag structure. 

It will output the same tag structure with the point version incremented by 1.